pkg/status/templates/logsagent.tmpl

```diff
@@ -32,7 +32,7 @@ Logs Agent
 
 {{- if .process_file_stats }}
   {{- range $metric_name, $metric_value := .process_file_stats }}
-    {{$metric_name}}: {{$metric_value}}
+    {{$metric_name}}: {{printf "%.0f" $metric_value}}
```

Comment:
If `process_file_stats` values are now integers, using floating point format here would produce error in the output: https://go.dev/play/p/LdX4DTpLiaI

Reply:
but why use the `printf`value if we're passing `uint64` now ?

Reply:
We initially pass `uint64`s to the `Status` struct in `pkg/log/status/status.go` because they came as `uint64`when we retrieved them from `gopsutil`'s `Process` class, but by the time the status command code is looking at the `logsagent.tmpl`, the variables seem to get converted to `float64`. If they remained `uint64`, we wouldn't have needed the `printf` statement.